Container tests not working on main

The PRs I put up are seeing failure unrelated to the changes in the following tests:
```
FAILED tests/test_containers.py::test_run_existing_container - aiodocker.exceptions.DockerError: [400] invalid JSON: json: cannot unmarshal string into Go struct field CreateRequest.Config.Entrypoint of type []string
FAILED tests/test_containers.py::test_run_container_with_missing_image - aiodocker.exceptions.DockerError: [400] invalid JSON: json: cannot unmarshal string into Go struct field CreateRequest.Config.Entrypoint of type []string
FAILED tests/test_containers.py::test_container_stats_list - aiodocker.exceptions.DockerError: [400] invalid JSON: json: cannot unmarshal string into Go struct field CreateRequest.Config.Entrypoint of type []string
FAILED tests/test_containers.py::test_container_stats_stream - aiodocker.exceptions.DockerError: [400] invalid JSON: json: cannot unmarshal string into Go struct field CreateRequest.Config.Entrypoint of type []string
```
I checked and it looks like the `docker:dind` image used in the test suite was updated 3 days ago: https://hub.docker.com/_/docker . And a version is not pinned in the the docker compose setup used by the tests:
https://github.com/aio-libs/aiodocker/blob/ef0c18cdaba44703254bc287cb62c6a3b62da4a7/tests/docker-compose.dind.yml#L5

So I'm thinking this is a failure related to update of that image.

--- a/CHANGES/1004.bugfix
+++ b/CHANGES/1004.bugfix
@@ -1,0 +1,1 @@
+Fix entrypoint usage in broken container tests to unblock CI.

--- a/tests/test_containers.py
+++ b/tests/test_containers.py
@@ -30,7 +30,7 @@ async def test_run_existing_container(docker: Docker, image_name: str) -> None:
     container = await docker.containers.run(
         config={
             "Cmd": ["-c", "print('hello')"],
-            "Entrypoint": "python",
+            "Entrypoint": ["python"],
             "Image": image_name,
         }
     )
@@ -56,7 +56,7 @@ async def test_run_container_with_missing_image(
     container = await docker.containers.run(
         config={
             "Cmd": ["-c", "print('hello')"],
-            "Entrypoint": "python",
+            "Entrypoint": ["python"],
             "Image": image_name,
         }
     )
@@ -125,7 +125,7 @@ async def test_container_stats_list(docker: Docker, image_name: str) -> None:
     container = await docker.containers.run(
         config={
             "Cmd": ["-c", "print('hello')"],
-            "Entrypoint": "python",
+            "Entrypoint": ["python"],
             "Image": image_name,
         }
     )
@@ -145,7 +145,7 @@ async def test_container_stats_stream(docker: Docker, image_name: str) -> None:
     container = await docker.containers.run(
         config={
             "Cmd": ["-c", "print('hello')"],
-            "Entrypoint": "python",
+            "Entrypoint": ["python"],
             "Image": image_name,
         }
     )

